packages/opc-feedback/src/assets/good.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 40 40"><path d="M20,0A20,20,0,1,0,40,20,20.023,20.023,0,0,0,20,0Zm0,38.261A18.261,18.261,0,1,1,38.261,20,18.281,18.281,0,0,1,20,38.261Z"/><path d="M101.227,130.613a2.613,2.613,0,1,0-2.613,2.613A2.617,2.617,0,0,0,101.227,130.613Zm-3.484,0a.871.871,0,1,1,.871.871A.873.873,0,0,1,97.742,130.613Z" transform="translate(-85.568 -114.091)"/><path d="M226.613,128a2.613,2.613,0,1,0,2.613,2.613A2.617,2.617,0,0,0,226.613,128Zm0,3.485a.871.871,0,1,1,.871-.871A.873.873,0,0,1,226.613,131.485Z" transform="translate(-199.659 -114.091)"/><path d="M121.132,224.222a.871.871,0,0,0-1.23.068,9.691,9.691,0,0,1-14.382,0,.871.871,0,0,0-1.3,1.162,11.433,11.433,0,0,0,16.978,0A.872.872,0,0,0,121.132,224.222Z" transform="translate(-92.711 -199.658)"/></svg>
```

Comment:
@hybridx can you confirm if we are adding assets to the individual component folders?

Reply:
This should be outside the `/src` folder